verifiersdemo/pom.xml

```diff
@@ -102,6 +102,281 @@
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKCopying.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKGraphRequest.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKProfilePictureView.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKAppLinkUtility.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKMacros.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKUtility.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKCoreKit.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKMutableCopying.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKAccessToken.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKGraphRequestDataAttachment.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKGraphRequestConnection.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKButton.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKSettings.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKTestUsersManager.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKAppEvents.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKGraphErrorRecoveryProcessor.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/FBSDKCoreKit.framework/Headers/FBSDKConstants.h
+                        </exclude>
+                        <exclude>
+                            /source/objc/EndpointOwnershipDemo.xcworkspace/contents.xcworkspacedata
```

Comment:
We can discuss it locally and provide a fix in the separate PR. Let's accept this now as a workaround. It is some sort of blocker.